src/App.js

```diff
@@ -114,7 +114,7 @@ function Artist({ artist, onSelect, onRemove, selectedListLength }) {
 
     }
       {show && <Card style={{ borderRadius: '8%', boxShadow: '10 10px 15px 8px rgba(0,0,0,0.06)' }}>
-        <Card.Img variant="top" src={artist.artist_img} style={{ borderRadius: '8%' }} />
+        <Card.Img variant="top" src={artist.artist_img} style={{ borderRadius: '8%', height: '16rem' }} />
```

Comment:
Good catch to include height change for both display options of cards in the if else statement